bot/exts/utilities/bookmark.py

```diff
@@ -23,6 +23,39 @@
 )
 
 
+class BookmarkTitleSelectionForm(discord.ui.Modal):
+    """
+    The form where a user can fill in a custom title for their bookmark & submit it.
+
+    This form is only available when the command is invoked from a context menu.
+    """
+
+    bookmark_title = discord.ui.TextInput(
+        label="Choose a title for you bookmark (optional)",
```

Comment:
```suggestion
        label="Choose a title for your bookmark (optional)",
```

Reply:
Fixed in https://github.com/python-discord/sir-lancebot/pull/1211/commits/5d0ac27ed0432359ee339382255eeb71a88f4b81 (https://github.com/python-discord/sir-lancebot/pull/1211)